cloudinit/config/cc_ca_certs.py

```diff
@@ -198,26 +246,37 @@ def handle(
     # If there is a remove_defaults option set to true, remove the system
     # default trusted CA certs first.
     if "remove-defaults" in ca_cert_cfg:
-        log.warning(
+        LOG.warning(
             "DEPRECATION: key 'ca-certs.remove-defaults' is now deprecated."
             " Use 'ca_certs.remove_defaults' instead."
         )
-        if ca_cert_cfg.get("remove-defaults", False):
-            log.debug("Removing default certificates")
-            remove_default_ca_certs(cloud.distro.name, distro_cfg)
+        if cloud.distro.name in ["fedora", "rhel"]:
```

Comment:
again, i would use family here

Reply:
Again, not sure if all the various RHEL derivatives use the same NSS utils for managing certificates. If so, then yes in general cc_ca_certs could have all the rhel family added as supported.

Reply:
aye

Reply:
Removed Fedora and RHEL related changes from this PR.